pkg/daemon/config.go

```diff
@@ -92,3 +106,55 @@ func tryToLoadOldConfig(nodeProfilesJson []byte) ([]ptpv1.PtpProfile, bool) {
 
 	return []ptpv1.PtpProfile{*ptpConfig}, true
 }
+
+func (conf *ptp4lConf) populatePtp4lConf(config *string) error {
```

Comment:
If the user only configure interfaces in the conf (with no global section) we need to create a global I think from the file inside the container

Reply:
Just a thought , we could use RegEx to fetch all interfaces 
```
sectionHead := regexp.MustCompile(`\[([^\[\]]*)\]`)
	matches := sectionHead.FindAllStringSubmatch(*p.Ptp4lConf, -1)
	for _, v := range matches {
		if v[1] != ignorePtp4lSection && v[1] != singleInterface {
			interfaces = append(interfaces, &v[1])
		}
	}
	```

Reply:
ignorePtp4lSection="[gloabl]"